src/containers/Storage/Storage.scss

```diff
@@ -19,4 +19,13 @@
     &__groups-wrapper {
         padding-right: 20px;
     }
+
+    &__controls {
+        width: unset;
+        margin-right: -40px;
+        padding-right: 40px;
+        padding-left: 20px;
+
+        transform: translateX(-20px);
```

Comment:
Could you please explain how it works and where 40 and 20 px come from?
mb move them to css variables for clearance

Reply:
In short, it's black magic :)
If serious, we have our scroll container `ydb-cluster` with side paddings. And thats the trouble. So, I just mislead this paddings (we have similar styles for tabs). Maybe we should rework layout for all cluster page, what do u think?

Reply:
I think if current solution fixes current problem - we could merge it but as well we could create task to techdebt with description of what should be reworked

I dont think for now we have that much of a resources for massive reworks of that type

However sometimes it could be done

I checked this issue with many pdisks and vdidks with mocks from this pr https://github.com/ydb-platform/ydb-embedded-ui/pull/1793 and it seems to work